MOBILE_CLIENT/COMMON/_HIGH/CORS_MISCONFIGURATION/recommendation.md

```diff
@@ -0,0 +1,72 @@
+
+# Cors Misconfiguration
+
+To mitigate `CORS` misconfiguration vulnerabilities, it is important to follow best practices. This includes properly configuring the Access-Control-Allow-Origin header to only allow trusted origins, rather than using the wildcard (*) value. Additionally, it is crucial to implement proper authentication and authorization mechanisms to ensure that only authorized users can access sensitive resources. Regularly monitoring and auditing `CORS` configurations can help identify and address any potential misconfigurations or vulnerabilities.
+
+# Code Examples:
+
+=== "Django"
+
+    ```Python
+        CORS_ALLOWED_ORIGINS = [
+        "https://cross-origin-website.com",
+        "https://sub.cross-origin-website.com",
```

Comment:
Does it take the scheme as well?

Reply:
you can instead of passing full links pass a regex pattern